### PR TITLE
layout: document the contents of a source image

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: layout.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+The list of a single manifest points to digest `sha256:60632c06e3ff9637fd5e5feb3f8768590fc94f9a701f99bc88a8f9e07c737b71`.
+_Notice_: We are overloading the use of the mediaType `application/vnd.oci.image.manifest.v1+json` so that the nested list of "`layers`" objects are nicely handled by the endpoint registry's garbage-collection.
+Here we include "`annotations`" denoting both the image tag, as well as an arbitrary type to indicate the type of this image.
+Using a "type" in this way is a complement pattern to the "platform.os" of a runnable container image.
+
+### manifest
+
+Using the digest pointed to in the image-index, is the [OCI manifest](https://github.com/opencontainers/image-spec/blob/v1.0.1/manifest.md) for this source image.
+Generally these manifest are geared for runnable container images, that will be fetched and composed from layers of file systems.
+For this first version of source container images, the layers are file systems, but that container the source objects that a corresponding runnable image are comprised of.
```
Comment (twaugh) [inline layout.md]:
```suggestion
For this first version of source container images, the layers are file systems, but that contain the source objects that a corresponding runnable image are comprised of.
```

File: layout.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+Join the OCI weekly discussions, and follow the [OCI Artifacts](https://github.com/opencontainers/artifacts) for possibilities.
+
+### Blobs
+
+In the first version of this source image approach, each of the "layer" blobs is a tar archive, as most container runtimes and registries expect.
+
+During this source image version that layers are TAR archives, the format of their contents are as follows:
+
+* a "blobs" top-level directory
+  * the source object itself is stored in an [blob digest](https://github.com/opencontainers/image-spec/blob/v1.0.1/descriptor.md#digests) directory structure
```
Comment (twaugh) [inline layout.md]:
```suggestion
  * the source object itself is stored in a [blob digest](https://github.com/opencontainers/image-spec/blob/v1.0.1/descriptor.md#digests) directory structure
```

File: layout.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+#### Next generation config
+
+Ideally this "config" will be document like a software bill of materials (SBOM).
+Join the OCI weekly discussions, and follow the [OCI Artifacts](https://github.com/opencontainers/artifacts) for possibilities.
+
+### Blobs
+
+In the first version of this source image approach, each of the "layer" blobs is a tar archive, as most container runtimes and registries expect.
+
+During this source image version that layers are TAR archives, the format of their contents are as follows:
```
Comment (twaugh) [inline layout.md:179]: This is what I was hoping for! :100:

**Review by twaugh — COMMENTED** [review]
Nice!

File: layout.md (hunk 3 above)
Comment (vbatts) [reply layout.md:179]: :smile_cat:

**Comment by vbatts** [comment]
addressed the couple of nits. PTAL

**Comment by twaugh** [comment]
LGTM

**Review by vrothberg — APPROVED** [review]
LGTM

**Pull request merged**